openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -405,8 +409,8 @@ def verify_error_response(field_name, data):
                 "Field '{0}' cannot be edited.".format(field_name), data["field_errors"][field_name]["user_message"]
             )
 
-        for field_name in ["username", "date_joined", "is_active"]:
-            response = self.send_patch(client, {field_name: "will_error", "gender": "f"}, expected_status=400)
+        for field_name in ["username", "date_joined", "is_active", "profile_image", "requires_parental_consent"]:
+            response = self.send_patch(client, {field_name: "will_error", "gender": "o"}, expected_status=400)
```

Comment:
@cahrens I made that change by inspection, so didn't think through that the default was still "m". I'll leave it as it is less confusing for the reader who might also expect that the default be "f".